deploy/000_deploy_deployer.ts

```diff
@@ -8,38 +8,50 @@ import { getDeploymentConfig } from './deployment.config'
 import { utils, Wallet } from 'zksync-web3'
 import { Deployer } from '@matterlabs/hardhat-zksync-deploy'
 
-const account = process.env.PRIVATE_KEY || ''
+const account = '0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110' // process.env.PRIVATE_KEY || ''
```

Comment:
Better back to ENV, already i'm having using that with my PK to deploy in testnet.